tests/unit/tools/test_mcp_config_builder.py

```diff
@@ -237,40 +214,52 @@ def test_dump_client_config_with_mixed_transports():
         assert result["openshift"]["transport"] == "stdio"
         assert result["sse-server"]["transport"] == "sse"
         assert result["openshift"]["env"]["OC_USER_TOKEN"] == user_token
-        assert (
-            result["sse-server"]["headers"][K8S_AUTH_HEADER] == f"Bearer {user_token}"
-        )
 
     @staticmethod
     def test_dump_client_config_with_streamable_http():
         """Test dump_client_config with streamable HTTP configuration."""
-        mcp_server_configs = [
-            MCPServerConfig(
-                name="streamable-server",
-                transport="streamable_http",
-                streamable_http=StreamableHttpTransportConfig(
-                    url="https://example.com/stream",
-                    headers={"X-Custom-Header": "value"},
-                    timeout=30,
-                    sse_read_timeout=60,
+        file_descriptor, file_path = tempfile.mkstemp(suffix=".tmp")
+        try:
+            with os.fdopen(file_descriptor, "w") as open_file:
+                open_file.write("value")
+            mcp_server_configs = [
+                MCPServerConfig(
+                    name="streamable-server",
+                    transport="streamable_http",
+                    streamable_http=StreamableHttpTransportConfig(
+                        url="https://example.com/stream",
+                        headers={
+                            "X-Custom-Header": file_path,
+                            "kubernetes": KUBERNETES_PLACEHOLDER,
```

Comment:
normally this header is "kubernetes-authorization" for kubernetes-mcp-server. anyway, it is a test, not blocking.